src/pkg/caendr/caendr/models/error.py

```diff
@@ -114,12 +114,10 @@ class UnprocessableEntity(InternalError):
 
 
 
-class CachedDataError(InternalError):
-  description = "This data has already been submitted by another user"
-  def __init__(self, report):
-    self.report = report
-    if report is not None and hasattr(report, 'data_hash'):
-      self.description = f'This data (hash {getattr(report, "data_hash")}) has already been submitted by another user'
+class JobAlreadyScheduledError(InternalError):

```

Comment:
I'm renaming this error type as part of the explicit separation between job creation & job submission.  Loosely speaking:
- If the user is trying to start a new job with data they've already submitted, that's a creation "error"
- If the user is trying to submit data that is already being computed (or already has results), that's a submission "error"